### PR TITLE
parsing item categories to match indexer logic

### PR DESCRIPTION
**Pull request by drewctate**
ISSUES CLOSED: #385

### CODE REVIEW

File: packages/content/src/portal.ts [hunk 1]
```diff
@@ -80,6 +81,7 @@ export function itemToContent(item: IItem): IHubContent {
     hubId: item.id,
     hubType: getItemHubType(item),
     normalizedType: normalizeItemType(item),
+    categories: parseItemCategories(item.categories),
```
Comment (tomwayson) [inline packages/content/src/portal.ts:84]: Let's not overwrite the item's own categories, but instead add a new property like 

```suggestion
    flattenedCategories: parseItemCategories(item.categories),
```

Open to suggestions on the name. Whatever we decide, let's add it as an optional (w/ `?`) property on `IHubCategory`.

**Review by tomwayson — COMMENTED** [review]
Nice work, just need to put it under a property w/ a different name.

File: packages/content/src/portal.ts (hunk 1 above)
Comment (drewctate) [reply packages/content/src/portal.ts:84]: I thought about this, too. The reason I overwrote categories is that that's what the indexer does. Will assigning it to a new property name maintain parity between content coming from the Hub and Portal APIs?

Comment (tomwayson) [reply packages/content/src/portal.ts:84]: Does the indexer preserve the item's categories under any other field (like `itemCategories`)?

Comment (tomwayson) [reply packages/content/src/portal.ts:84]: If so, let's duplicate that behavior here.

If not, let's do that here on the client and then open an issue in the indexer repo to talk about doing the same there.

Comment (drewctate) [reply packages/content/src/portal.ts:84]: Okay, it looks like it doesn't. `flattenedCategories` sounds good. I'll open an issue on the indexer.

Comment (tomwayson) [reply packages/content/src/portal.ts:84]: well in this case we would not be adding `flattenedCategories`, we'd be adding `itemCategories = item.categories` in order to not create a breaking change.

Comment (drewctate) [reply packages/content/src/portal.ts:84]: I see. Okay. Worth noting that unfortunately, `itemCategories` will only be genuine when we're using the Portal API. For content from the Hub API, the two attributes will be identical.

**Review by tomwayson — CHANGES_REQUESTED** [review]
One more small fix.

**Review by tomwayson — APPROVED** [review]
Well done.

**Pull request merged**